### DISCUSSION TITLE
Route timeout in CARLA autonomous driving leaderboard

### DISCUSSION THREAD

**Discussion by violasox** [carla-simulator/carla#5499] [Q&A]
I've been trying to understand how long agents have to complete routes in the [AD leaderboard](https://leaderboard.carla.org/). After some digging through the code, I found the SECONDS_GIVEN_PER_METERS parameter in [route_scenario.py](https://github.com/carla-simulator/leaderboard/blob/111a48f9099c08a2f1068ee8aea2ad56ce52ef9d/leaderboard/scenarios/route_scenario.py). Since this value is set to 0.8s, that implies that the slowest speed a vehicle can drive on average and avoid a route timeout is 1m/0.8s = 4.5km/hr. However, the durations on the leaderboard entries seem to imply much slower speeds than this. For example, the top leaderboard entry takes 4794 min, which for 173km of evaluation implies 2.2km/hr. The 2nd place entry is even worse, taking 23665 min -> 0.4km/hr.

Something clearly doesn't add up in the documentation or my understanding! I would appreciate any help.

**Comment by ajdroid** [comment]
Did you ever get an answer for this? If not here, I might suggest the Carla discord where you might receive a reply to this kind of question
